api/alert_channels_test.go

```diff
@@ -238,6 +238,44 @@ func generateAlertChannels(guids []string, iType string) string {
 	return strings.Join(alertChannels, ", ")
 }
 
+func TestAlertChannelsTest(t *testing.T) {
+	var (
+		intgGUID   = intgguid.New()
+		apiPath    = fmt.Sprintf("AlertChannels/%s/test", intgGUID)
+		fakeServer = lacework.MockServer()
+	)
+	fakeServer.UseApiV2()
+	fakeServer.MockToken("TOKEN")
+	defer fakeServer.Close()
+
+	fakeServer.MockAPI(apiPath,
+		func(w http.ResponseWriter, r *http.Request) {
+			if assert.Equal(t, "POST", r.Method, "Post() should be a POST method") {
```

Comment:
I think you meant to say `Test()` should be `POST`
```suggestion
			if assert.Equal(t, "POST", r.Method, "Test() should be a POST method") {
```